contribution/testing_guide.md

```diff
@@ -56,16 +56,10 @@ $ kubectl -n [namespace name] exec -it [pod name] -- bash -c [command]
 
 ## 1.6. Check generated alerts
 
-- Log file
+- Log using [karmor](https://github.com/kubearmor/kubearmor-client) cli tool
 
     ```text
-    $ tail (-f) /tmp/kubearmor.log
-    ```
-
-    If you changed the location of a log file, replace the default file path to your file.
-
-    ```text
-    $ tail (-f) [your log file path]
+    $ tail |  karmor log --json
```

Comment:
We don't need to tail, `karmor log` will only show future generated alerts not any previous ones.
```suggestion
    $ karmor log [options....]
```
Followed up by available options.

Also I think we can remove the log client part, since karmor handles it.

Reply:
Hmm, yeah that's correct.